jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/MenusTests.java

```diff
@@ -33,7 +33,11 @@
 
 public class MenusTests extends TestsInit {
 
-    List<String> optionsTitles = Arrays.asList("Click Me", "Click Me", "Click Me", "Click Me 2");
+    private static final List<String> OPTIONS_TITLES_NAMES = Arrays.asList("John Leider", "Enable messages", "Enable hints");
```

Comment:
Please, check Natalia's commits. Looks like shi fixed MenusTests. But I may be wrong :) Thanx